CONTRIBUTING.md

```diff
@@ -72,3 +72,32 @@ To add a new test we recommend to run the `inframap prune --canonicals --tfstate
 #### Add a new Printer
 
 To add a new Printer just follow the `printer.Printer` interface and add it to `printer/{printer_name}` and `printer/type.go` and run `make generate`. After that it'll be directly added to the list of available `--printer`.
+
+#### Add new icons
+
+The Icons are taken from the mapping we have on [tfdocs](https://github.com/cycloidio/tfdocs), but instead of `.svg` we use `.png` and they are place in `assets/{provider}` with a size of 72x72.
```

Comment:
```suggestion
Icons follow the mapping defined in [tfdocs](https://github.com/cycloidio/tfdocs). Instead of `.svg` we use `.png` and files are located under the `assets/{provider}` directory with a size of 72x72.
```